### ISSUE TITLE
Please keep min php version at least at 7.0

### ISSUE THREAD

**Issue by kremsy** [joeldg/bowhead#37]
On debian machines its almost impossible to catch up to 7.1 without changing to some special packages, just upgraded my php for the Project from 5.6 to 7.0 on debian8 which was already quite a mess. Would be amazing if 7.0 could be supported and stay in support, should not make a big coding difference anyways. (PHP Storm offers a nice Setting for the Language level, so you can't mistakly use Syntax from above).

**Comment by dhensen** [comment]
You could point PHPStorm to 7.1 and the run the source exclusively in docker where 7.1 is installed.

**Comment by teneon** [comment]
Ubuntu 16.04 (the official long term release) officially also supports only php 7.0. Upgrading to 7.1 is quite messy :/ Is bowhead working on 7.0 or 7.1 is a must?

**Comment by kremsy** [comment]
Yep, for debian its even more complicate, only had problems in 1-2 bowhead files if i remember right, they are not 7.0 compatible but as a code you can get away around it^^

**Comment by joeldg** [comment]
It should work on 7.0, I do make use of 7.0 specific items though so there is no way it will run on anything lower.

For 7.1: I will probably be using Symmetric array destructuring and pseudo-types for some of the front end which is a 7.1 and Multi catch exception handling in the data collectors will probably happen.

I will be steaming on to 7.2 as well because of Sodium and the trailing comma thing and some of the new socket stuff could help with FIX support that I have been looking to add. But, I can probably keep it at 7.1 pretty easy.

It's weird you would have issues with debian, just add ppa:ondrej/php, update and poke around the packages with apt-cache search php.. Or did you mean your project you are running has an issue. If so, just run the docker container ```docker pull joeldg/bowhead```

**Comment by kremsy** [comment]
The problem arent any packages or whatever on debian, the problem is that debian itself support never the newest version, debian 7 supports 5.6 max, debian 8 7, they moving way slower as its using only very stable packages. Kinda the same as you cant use java 8 apps on old smartphones^^.

**Comment by kremsy** [comment]
https://wiki.debian.org/PHP#Available_versions

**Comment by kilo42L** [comment]
Yep im having same issue..debian
